jena-arq/src/main/java/org/apache/jena/riot/adapters/AdapterFileManager.java

```diff
@@ -285,6 +286,12 @@ protected Model readModelWorker(Model model, String filenameOrURI, String baseUR
         if ( baseURI == null )
             baseURI = SysRIOT.chooseBaseIRI(filenameOrURI) ;
         try(TypedInputStream in = streamManager.openNoMapOrNull(mappedURI)) {
+            if ( in == null )
+            {
+                if ( log.isDebugEnabled() )
+                    log.debug("Failed to locate '"+mappedURI+"'") ;
```

Comment:
As I understand it, `log.debug("Failed to locate '{}'", mappedURI) ` [avoids the need](https://www.slf4j.org/faq.html#logging_performance) to explicitly check `isDebugEnabled()`.

Reply:
Yes, I just copied and pasted this from the parent class where the same logic is present

Reply:
More recently, `FmtLog.debug(logger, fmtString, args...)`.